### PR TITLE
golint fixed for parts of routers root, dev, user and org dirs

### PR DESCRIPTION
**Pull request by lunny**

### CODE REVIEW

**Review by tboerger — APPROVED** [review]

**Comment by tboerger** [comment]
LGTM

**Comment by appleboy** [comment]
👍

**Comment by andreynering** [comment]
Travis failed

File: routers/dev/template.go [hunk 1]
```diff
@@ -11,6 +11,7 @@ import (
 	"code.gitea.io/gitea/modules/setting"
 )
 
+// TemplatePreview render only on DEV mode for preview the template
```
Comment (bkcsoft) [inline routers/dev/template.go]: This function always renders if asked, `RUN_MODE` is checked elsewhere

File: routers/home.go [hunk 2]
```diff
@@ -42,9 +47,10 @@ func Home(ctx *context.Context) {
 	}
 
 	ctx.Data["PageIsHome"] = true
-	ctx.HTML(200, HOME)
+	ctx.HTML(200, TplHome)
 }
 
+// RepoSearchOptions options when calling search repositories
```
Comment (bkcsoft) [inline routers/home.go]: Maybe drop the additional `options` as it's redundant?

File: routers/home.go [hunk 3]
```diff
@@ -125,6 +134,7 @@ type UserSearchOptions struct {
 	TplName  base.TplName
 }
 
+// RenderUserSearch render user search page
```
Comment (bkcsoft) [inline routers/home.go:137]: This could just be `RenderUserSearch page` :)

File: routers/home.go [hunk 4]
```diff
@@ -17,17 +17,22 @@ import (
 )
 
 const (
-	HOME                  base.TplName = "home"
-	EXPLORE_REPOS         base.TplName = "explore/repos"
-	EXPLORE_USERS         base.TplName = "explore/users"
-	EXPLORE_ORGANIZATIONS base.TplName = "explore/organizations"
+	// TplHome home page template
+	TplHome base.TplName = "home"
```
Comment (bkcsoft) [inline routers/home.go]: Should these really be exported? :confused:

File: routers/install.go [hunk 5]
```diff
@@ -34,7 +34,8 @@ import (
 )
 
 const (
-	INSTALL base.TplName = "install"
+	// TplInstall template for installation page
+	TplInstall base.TplName = "install"
```
Comment (bkcsoft) [inline routers/install.go]: Exported?

File: routers/org/members.go [hunk 6]
```diff
@@ -15,10 +15,13 @@ import (
 )
 
 const (
-	MEMBERS       base.TplName = "org/member/members"
-	MEMBER_INVITE base.TplName = "org/member/invite"
+	// TplMembers template for organization members page
+	TplMembers base.TplName = "org/member/members"
```
Comment (bkcsoft) [inline routers/org/members.go]: Exported?

File: routers/org/org.go [hunk 7]
```diff
@@ -14,19 +14,22 @@ import (
 )
 
 const (
-	CREATE base.TplName = "org/create"
+	// TplCreateOrg template path for create organization
+	TplCreateOrg base.TplName = "org/create"
```
Comment (bkcsoft) [inline routers/org/org.go]: Exported?

File: routers/org/setting.go [hunk 8]
```diff
@@ -17,23 +17,28 @@ import (
 )
 
 const (
-	SETTINGS_OPTIONS base.TplName = "org/settings/options"
-	SETTINGS_DELETE  base.TplName = "org/settings/delete"
-	SETTINGS_HOOKS   base.TplName = "org/settings/hooks"
+	// TplSettingsOptions template path for render settings
+	TplSettingsOptions base.TplName = "org/settings/options"
```
Comment (bkcsoft) [inline routers/org/setting.go]: Exported?

File: routers/org/setting.go [hunk 9]
```diff
@@ -17,23 +17,28 @@ import (
...
+	// TplSettingsDelete template path for render delete repository
+	TplSettingsDelete base.TplName = "org/settings/delete"
+	// TplSettingsHooks template path for render hook settings
+	TplSettingsHooks base.TplName = "org/settings/hooks"
 )
 
+// Settings render the main settings page
```
Comment (bkcsoft) [inline routers/org/setting.go:28]: `organization main settings page` ?

File: routers/org/teams.go [hunk 10]
```diff
@@ -17,12 +17,17 @@ import (
 )
 
 const (
-	TEAMS             base.TplName = "org/team/teams"
-	TEAM_NEW          base.TplName = "org/team/new"
-	TEAM_MEMBERS      base.TplName = "org/team/members"
-	TEAM_REPOSITORIES base.TplName = "org/team/repositories"
+	// TplTeams template path for teams list page
+	TplTeams base.TplName = "org/team/teams"
```
Comment (bkcsoft) [inline routers/org/teams.go]: Exported?

**Review by bkcsoft — CHANGES_REQUESTED** [review]

**Comment by lunny** [comment]
@bkcsoft Please review again.

**Review by bkcsoft — COMMENTED** [review]
Don't really care, but worth mentioning

**Review by bkcsoft — APPROVED** [review]

**Comment by bkcsoft** [comment]
LGTM

**Comment by lunny** [comment]
resolved #70

**Pull request merged**